adminservice/src/intTest/groovy/nu/yona/server/LocalizationTest.groovy

```diff
@@ -1,56 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2015 Stichting Yona Foundation
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v.2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at https://mozilla.org/MPL/2.0/.
- *******************************************************************************/
-package nu.yona.server
-
-import groovy.json.*
-import spock.lang.Shared
-import spock.lang.Specification
-
-class LocalizationTest extends Specification
```

Comment:
Given that the admin service now returns a localizable name rather than a localized one, this test does not make sense anymore. There is still another ``LocalizationTest`` for the app service that test the localization there.